.gitignore

```diff
@@ -36,3 +36,5 @@ liburing.pc
 liburing-ffi.pc
 
 cscope.out
+
+/.vscode
```

Comment:
I don't think commit a6dc05bdd2e7362160cb4427285a67e91c954fe3 ("chore: Add `.editorconfig` to maintain consistent coding styles for different editors") makes sense to be included in liburing upstream. There are hundreds of text editors out there.

You add vscode specific changes. It's not relevant to liburing, and it's a personal text editor choice, not something you should bring upstream.

Reply:
Removed